core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/WorkspaceComponent.scala

```diff
@@ -370,8 +376,18 @@ trait WorkspaceComponent {
       filter(rec => (rec.namespace === namespaceName.value))
     }
 
+    def listOutdatedEntityCaches(limit: Int): ReadAction[Seq[(UUID, Timestamp)]] = {
+      //We don't necessarily want all of the workspaces back, just the ones that we can handle in one pass
+      //We also order by lastModified so we can handle the ones that are the longest out of date
+      filter(rec => (rec.entityCacheLastUpdated < rec.lastModified)).take(limit).sortBy(_.lastModified.asc).map { ws => (ws.id, ws.lastModified) }.result
```

Comment:
There are a few cases where lastModified may be updated even when entities aren't updated. All of the cases are:

* Updating entities directly
* A submission completing (may up date entities, might not- hard to know)
* A workspace attribute being updated
* A method config being updated

I was on the fence about it being worth it to make this more specific, but it might be worth addressing to reduce any potential load. I think those events are probably lower volume, however.